src/components/Map.js

```diff
@@ -370,7 +370,7 @@ export default function MapContainer(props) {
               </Circle>
             );
           })}
-        {Array.isArray(internationalData) &&
+        {/* {Array.isArray(internationalData) &&
```

Comment:
Any specifc reason for commenting this out?

Reply:
there was a production issue because of this code. The API is not returning international data.